### DISCUSSION TITLE
Footnotes & bibliography in text block

### DISCUSSION THREAD

**Discussion by ktal4** [elementor/elementor#22147] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I'm an academic and my blog posts often require footnotes, but it's hard to make them in Elementor. I'm sure I'm not the only academic who has this issue (though I searched the feature requests and didn't see one).

### Describe the solution you'd like

It would be nice if we could pull references from Zotero, Mendeley, etc. and insert them in block text and generate a bibliography for the end of the post.  Maybe an "academic text" block?

### Describe alternatives you've considered

Right now I just put the numbers or citations in as superscript or in parentheses, and generate the bibliography in a separate text block by hand.

**Comment by vgstef** [comment]
Yes! I'm facing the same problem. I often have clients who have articles/blog post that need some kind of footnote/reference.
